### ISSUE TITLE
brew-file no longer expands custom environment variables for brewfile includes

### ISSUE THREAD

**Issue by zcutlip** [rcmdnk/homebrew-file#109]
I'm not sure if this recent change was intentional or not, but previously I could define custom shell variables in my environment like:

```sh
export ARCH=$(uname -m)
```

And then include appropriate brewfiles using those variables:
```
Include architecture-specific brewfile
file ./Brewfile.arch_${ARCH}
``` 

This seems to no longer work for any custom environment variables I've set.

Another example is that worked until recently:
```sh
# unset this to not include "extra" brewfiles such as extra-casks.Brewfile
export HBB_EXTRA="extra"
```

Then include:
```
# include "extra" casks only on machines that opt in
file ./${HBB_EXTRA}-casks.Brewfile
```

Standard variables like `$HOSTNAME` still seem to work.

**Comment by rcmdnk** [comment]
Thank you for the report.

It seems it does not work in M1 Mac (arm64) environment.

On my Mac (M1 Mac), it works in Rosetta, x86_64 environment, but does not work in arm64 environment.
It also works in Ubuntu (with x86_64).

There is a something wrong in Python's `os.environ.get` function with arm64?

`HOSTNAME` is a shell variable and it is taken directly from python module.
Then it works even with arm64.

https://github.com/rcmdnk/homebrew-file/blob/e5b1556789f2e0ce1a5e5cb2e24ef281a5c825a4/bin/brew-file#L83


One question: Do you remember if it works even in arm64 previously?

**Comment by zcutlip** [comment]
In my case, the problem is happening on both of my Intel Macs. I haven't checked on my M1 Mac, but I assuming it's happening there as well.

**Comment by zcutlip** [comment]
I may have misunderstood the issue.

I ran `brew-file` in vscode, and single stepped in the debugger. It's definitely expanding `${ARCH}` and opening & reading `Brewfile.arch_x86_64`.

I'm not sure what the exact issue is, but what happens is the following:
- Any new entries in the included brewfile are ignored
- Any existing entries in the included brewfile that are already installed get added to the main brewfile
- The included `Brewfile.arch_${ARCH}` brewfile is not modified in any way, even if entries are moved to the main brewfile, or if it contains entries that are not installed

So even though, in the debugger, I can watch `brew-file` open the file and completely parse it, it doesn't seem to respect it for installing, uninstalling, and re-initializing.

**Comment by zcutlip** [comment]
Another data point:

After doing `brew file update`, I see the following in the output suggesting `${ARCH}` is not being expanded:

```

##########################################################
# Initialize /Users/zach/.dotfiles/brewfile/Brewfile.arch_
##########################################################
```

However, when I do `brew-file get_files`, I see that `${ARCH}` has been expanded:

```
brew-file get_files
/Users/zach/.dotfiles/brewfile/Brewfile
/Users/zach/.dotfiles/brewfile/Brewfile.arch_x86_64
```

**Comment by rcmdnk** [comment]
I think I understand.

The problem is that original `brew` command throws away environmental variables when it calls a subcommand.

I think you have used [brew-wrap](https://homebrew-file.readthedocs.io/en/latest/brew-wrap.html)
and it makes you to call `brew-file` directly if you call `brew file`.

But, I'm sorry, the implementation of brew-wrap at v8.5.5 was wrong
and you call `brew file` directly even if you set brew-wrap.

I fixed it in v8.5.6.

Could you please update brew-file and try it?

    $ brew update && brew upgrade
    $ source ~/.bashrc # or source ~/.zshrc # or directly, source /usr/local/etc/brew-wrap
    $ brew file cat

**Comment by zcutlip** [comment]
Great, this seems to work on v8.5.6!

Thank you
